src/oneStepOneAleaProblem.jl

```diff
@@ -93,12 +94,11 @@ function solve_one_step_one_alea(model, #::SDDP.LinearDynamicLinearCostSPmodel,
 
     if solved
         optimalControl = getValue(u)
-
         # Return object storing results:
         result = SDDP.NextStep(
                           [model.dynamics(xt, optimalControl, xi)],
                           optimalControl,
-                          getDual(m.ext[:cons][1]),
+                          [getDual(m.ext[:cons][i]) for i in 1:model.dimStates],
                           getObjectiveValue(m))
     else
         # If no solution is found, then return nothing
```

Comment:
I think that for the moment we can keep going with a fixed dimension of control and state but a TODO would be great :)